### PR TITLE
Adds MostRead to STY's second column

### LINKED ISSUE #5737: Add MostRead to STY's secondary column
**Is your feature request related to a problem? Please describe.**
We need to add the existing `MostRead` component to the secondary column of STY pages following on from this merge https://github.com/bbc/simorgh/pull/5417. 

**Describe the solution you'd like**
Re-use the existing `MostRead` container that the ws-home team has been working on and implement it into the secondary column of STY pages. Fix any layout/styling issues that appear with STY pages.

Work needs to be done to fix layout on Desktop breakpoints, this is currently being done in Psammead see https://github.com/bbc/psammead/issues/3178.

**Describe alternatives you've considered**

**Testing notes**
This will need a dev a11y swarm before being merged.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves #5737

**Overall change:** 
Added `MostRead` container to STY's second column.

**Code changes:**
- Bumped `psammead-most-read` to version 4.0.2.
- Bumped `psammead-grid` to version 1.1.11.
- Replaced `maxTwoColumns` boolean with new string prop from most-read.
- Removed background of STY component wrapper.
- Updated Snapshots.
- Added and updated Stories on storybook.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing -  Regression testing for Front Pages and Articles
- [x] Dev a11y swarm.

### CODE REVIEW

File: src/app/containers/MostRead/index.jsx [hunk 1]
```diff
@@ -46,14 +46,14 @@ const MostReadContainer = ({
...
-  maxTwoColumns: bool,
+  columnLayout: oneOf(['oneColumn', 'twoColumn', 'multiColumn']),
   isOnFrontPage: bool,
 };
 
 MostReadContainer.defaultProps = {
   mostReadEndpointOverride: null,
   constrainMaxWidth: false,
-  maxTwoColumns: false,
+  columnLayout: 'multiColumn',
```
Comment (EinsteinNjoroge) [inline src/app/containers/MostRead/index.jsx:56]: This is already defined here https://github.com/bbc/simorgh/pull/5739/files#diff-04298a8f3ec1cbafcb68a13991abd880R201, I think we can do away with this default prop definition and retain the implementation in `canonical/index.js`

File: src/app/containers/MostRead/index.jsx [hunk 2]
```diff
@@ -46,14 +46,14 @@ const MostReadContainer = ({
 MostReadContainer.propTypes = {
   mostReadEndpointOverride: string,
   constrainMaxWidth: bool,
-  maxTwoColumns: bool,
+  columnLayout: oneOf(['oneColumn', 'twoColumn', 'multiColumn']),
```
Comment (EinsteinNjoroge) [inline src/app/containers/MostRead/index.jsx:49]: Same as above.
https://github.com/bbc/simorgh/pull/5739/files#diff-04298a8f3ec1cbafcb68a13991abd880R196

**Review by EinsteinNjoroge — COMMENTED** [review]
Looks good, just a few suggestions ☝️

File: src/app/containers/MostRead/__snapshots__/index.test.jsx.snap [hunk 3]
```diff
@@ -6,7 +6,7 @@ exports[`MostReadContainerCanonical Assertion should render most read correctly
   margin: 0;
   padding: 0;
   grid-auto-flow: column;
-  grid-template-rows: repeat( 5,[col-start] auto [col-end] );
```
Comment (HarveyPeachey) [inline src/app/containers/MostRead/__snapshots__/index.test.jsx.snap:9]: A bit strange that it's changing this value from 5 to 10. Will have to make sure this is double-checked in a11y swarm and test phase.

File: src/app/containers/MostRead/index.jsx (hunk 2 above)
Comment (HarveyPeachey) [reply src/app/containers/MostRead/index.jsx:49]: @EinsteinNjoroge Wouldn't we still need to declare the props as they still need to be passed down to the canonical component?

**Comment by thekp** [comment]
~This issue: https://github.com/bbc/psammead/issues/3234 needs to be resolved before we can merge this PR.~ 
Resolved now.

File: src/app/containers/MostRead/index.jsx (hunk 2 above)
Comment (EinsteinNjoroge) [reply src/app/containers/MostRead/index.jsx:49]: Sure, this makes sense. We can leave it as is.

**Review by thekp — APPROVED** [review]
👍 LGTM 💯 This updates most read by 2 breaking changes, we should regression test most read on all page types where it's used (frontpage/optimo aritcle).

**Review by hotinglok — COMMENTED** [review]
https://github.com/bbc/simorgh/blob/22a2107dd0e995fb5862ee1556ad57f9fc5bca64/src/app/containers/MostRead/index.stories.jsx#L33
Need to update the stories as well, but looks good otherwise!

**Review by j-pendlebury — APPROVED** [review]
LGTM 👍

**Comment by HarveyPeachey** [comment]
**Dev/A11y Swarm**

-https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers
  - [x] Axe
  - [x] Design
  - [x] Colour Contrast
  - [x] Colour blindness
  - [x] Windows HCM
  - [x] FF Colour Preferences
  - [x] Text 200%, Page 200%
  - [ ] ~Icons~
  - [ ] ~Heading Order~
  - [x] Check accesibility of browser DOM
  - [x] Keyboard Navigation 

**_Screen Readers (IE11)_** 
- [x] ZoomText Magnifier/Reader
- [x] Dragon NaturallySpeaking
- [x] JAWS - **4.** "Mas Leidas" is read out in English then it reads out "Heading level 2 - Mas leidas" in a Spanish accent.
- [x] Read&Write - "Mas Leidas" is read out in English then it reads out "Heading level 2 - Mas leidas" in a Spanish accent.
- [x] NVDA (Firefox) - "Mas Leidas" is read out in English then it reads out "Heading level 2 - Mas leidas" in a Spanish accent.

**_Mac, iOS, and Android:_** 
- [x] VoiceOver iOS (iPhone/iPad)
"Mas Leidas" is read out in Spanish then it reads out "Heading level 2 - Mas leidas" in Spanish.
- [x] VoiceOver OS (Mac)
- [x] TalkBack (Chrome on Android)

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**